lib/taste_tester/config.rb

```diff
@@ -95,8 +95,13 @@ def self.relative_databag_dir
     end
 
     def self.chef_port
+      require 'taste_tester/state'
+      previous_ports = []
+      if TasteTester::State.port
+        previous_ports << TasteTester::State.port
+      end
       range = chef_port_range.first.to_i..chef_port_range.last.to_i
-      range.to_a.shuffle.each do |port|
+      (previous_ports + range.to_a.shuffle).each do |port|
```

Comment:
it took me a minute to figure out why you made that an array. FWIW, I woulda done it this way:

```
range = (chef_port_range.first.to_i..chef_port_range.last.to_i).to_a
if TasteTester::State.port
  range.prepend(TasteTester::State.port)
end
range.shuffle.each do |port|
```